packages/components/modal/src/modal.css

```diff
@@ -0,0 +1,67 @@
+/* General........................................................ */
+
+:host {
```

Comment:
Maybe removing the empty rule? there are couple of more of these around the codebase